lib/spack/docs/packaging_guide.rst

```diff
@@ -136,6 +136,10 @@ generates a boilerplate template for your package, and opens up the new
        homepage = "http://www.example.com"
        url      = "https://gmplib.org/download/gmp/gmp-6.1.2.tar.bz2"
 
+       # FIXME: Add a list of GitHub accounts to automatically
+       # notify when the package is updated.
+       # maintainers = ['foo', 'bar']
```

Comment:
These are actually real github users, and they'll be added as reviewers on newly opened PRs that just uncomment the FIXME's.  `github_user1` and `github_user2` are not taken -- let's use those.

Reply:
Should we grab the relevant accounts and put `spack_maintainer1`, etc.